streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStateUpdateTask.java

```diff
@@ -32,26 +38,33 @@
 /**
  * Updates the state for all Global State Stores.
  */
-public class GlobalStateUpdateTask implements GlobalStateMaintainer {
+public class GlobalStateUpdateTask implements GlobalTask {
```

Comment:
Please ignore the Global store changes so far: they are a bit partial and I may want to revert them all for this PR just for ease of reviewing.

Reply:
And other related GlobalXX classes too.